docker-compose.yml

```diff
@@ -55,3 +57,16 @@ services:
         retries: 6
     ports:
       - "1433:1433"
+
+  sql-data:
```

Comment:
is `sql-data` the best name? or `sql-mts`, or `sql-mts-data` maybe?

Reply:
changed name to sql-mts-data